apps/drain-safe/src/components/AddressInput.tsx

```diff
@@ -3,7 +3,7 @@ import { AddressInput } from '@gnosis.pm/safe-react-components';
 
 export default styled(AddressInput)`
   && {
-    width: 520px;
```

Comment:
Do we need to change this styles? Is on a different app

Reply:
yep sorry! 😓

Reply:
removed!
:3